api/apps/dataset_api.py

```diff
@@ -591,7 +607,142 @@ def download_document(dataset_id, document_id):
     except Exception as e:
         return construct_error_response(e)
 
-# ----------------------------start parsing-----------------------------------------------------
+# ----------------------------start parsing a document-----------------------------------------------------
+# helper method for parsing
+def dummy(prog=None, msg=""):
+    pass
+
+
+def doc_chunk(binary, name, parser_id, tenant_id):
```

Comment:
What is this function used to ? Use a better understanding name and write some comments?